### ISSUE TITLE
nvda does not announce first line indentation in libre office writer

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13052]
### Steps to reproduce:
open the free office writer.
type a paragraph with 2 lines
apply the effects

first line indent (change the default, I put 1.30)
Now read the applied formatting 
### Actual behavior:
nvda does not report the indentation of the first line

### Expected behavior:
nvda should report the indentation of the first line 

#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
NVDA version alpha-24184,fd9ae8c4
#### Windows version:
11 22000.318
#### Name and version of other software in use when reproducing the issue:
LibreOffice version 7.3.0.0
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by cyrmax** [comment]
November 17 2022, this issue still exists. 
I guess this problem cannot be fixed on NVDA side. The issue should be reported to Libre Office developers.

**Comment by seanbudd** [comment]
>I guess this problem cannot be fixed on NVDA side. 

Is there a reason for this assumption?
NV Access has not performed an investigation yet.
Reporting it to LibreOffice developers may help, but the issue could still be with NVDA as far as we are aware.

**Comment by cyrmax** [comment]
> Is there a reason for this assumption? NV Access has not performed an investigation yet. Reporting it to LibreOffice developers may help, but the issue could still be with NVDA as far as we are aware.


I have tested this behavior on LinuxMint and there are problems too. Orca reads randomly "on, off, on" and letters that i press along with ctrl key to set alignment, something like "e on off on" or "j off on". 

Also i know that in Microsoft Word such things like changes of alignment, tabulation, etc are reported to screen readers via UIA or something like this. 
So i thought that Libre Office should report information to NVDA for reading.

**Comment by fernando-jose-silva** [comment]
nº 6913 
appears to be duplicated from the right, centered alignment information.
the problem remained for first line indentation

**Comment by Adriani90** [comment]
cc: @michaelweghorn, it seems in MS Word it works as expected but not in LO 25.2.

**Comment by michaelweghorn** [comment]
>Looking at the [IAccessible2 object specification](https://github.com/LinuxA11y/IAccessible2/blob/master/spec/objectattributes.md) (and the referenced CSS spec), implementing support for the "text-indent" object property in both, LibreOffice and NVDA might be a way to make it work.

LO >= 25.8 now reports that attribute, see commit https://git.libreoffice.org/core/commit/a2d3c9341f177b720d7a52e70759721d3221037f

PR to implement the NVDA side: https://github.com/nvaccess/nvda/pull/17638
